src/stores/certificate-trust-api.ts

```diff
@@ -0,0 +1,24 @@
+import { createApi, fetchBaseQuery } from '@reduxjs/toolkit/query/react';
+import { getIpcApi } from 'src/lib/get-ipc-api';
+
+export const certificateTrustApi = createApi( {
+	reducerPath: 'certificateTrustApi',
+	baseQuery: fetchBaseQuery(),
+	tagTypes: [ 'CertificateTrust' ],
+	endpoints: ( builder ) => ( {
```

Comment:
```suggestion
	refetchOnFocus: true,
	endpoints: ( builder ) => ( {
```
I realize now that we can configure this at the global level and avoid some repetition in the components.

Reply:
I have added it and it works like a magic. 👍